src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -686,7 +686,9 @@ export function checkCommunityCareEligibility() {
         return response.eligible;
       }
     } catch (e) {
-      captureError(e);
+      captureError(e, false, null, {
+        facilities: state.user?.profile?.facilities,
```

Comment:
We're getting some TypeErrors in Sentry that seem to indicate this value isn't an array. I can't figure out why or how that would happen, so I'm curious what the value is when those errors happen.